htdocs/compta/bank/annuel.php

```diff
@@ -195,10 +195,9 @@
 for ($mois = 1; $mois < 13; $mois++) {
 	print '<tr class="oddeven">';
 	print "<td>".dol_print_date(dol_mktime(1, 1, 1, $mois, 1, 2000), "%B")."</td>";
+	$totsorties[$annee] = 0;
```

Comment:
Here variable $annee is not defined.

Reply:
you are right :
 for ($annee = $year_start; $annee <= $year_end; $annee++) {
                if ($mois == 1){
                        $totsorties[$annee] = 0;
                        $totentrees[$annee] = 0;
                }
                $case = sprintf("%04s-%02s", $annee, $mois);

same result with better code